packages/gatsby-plugin-google-tag-manager/src/pixel/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { Fragment, useMemo } from 'react'
+import type { FC } from 'react'
+import { isServer } from '@vtex/gatsby-theme-store/src/utils/env'
+import { useLazyScript } from '@vtex/gatsby-theme-store/src/sdk/lazyScript/useLazyScript'
+import { usePixelEvent } from '@vtex/gatsby-theme-store/src/sdk/pixel/usePixelEvent'
+import { once } from '@vtex/gatsby-theme-store/src/sdk/once'
+
+import { handler } from './handler'
+
+const DEFAULT_DATALAYER_CONFIG = [{ 'gtm.blacklist': ['html'] }]
```

Comment:
Maybe have an option to disable that too

Reply:
Nice! I will do that in a future contribution. #vemai